File: Java/bus.java
/*
 *	Problem: "Bus"
 */

import java.util.Scanner;

public class bus {
	
	/*-----mySolution-----*/
	public static void mySolution(Scanner sc, int cases) {
		for (int i = 0; i < cases; i++) {
			int k = sc.nextInt();
			int result = 0;
			for (int j = 0; j < k; j++) {
				result = (result*2) + 1;
			}
			System.out.format("%d\n", result);
		}
	}
	
	/*-----main-----*/
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int cases = sc.nextInt();
		mySolution(sc, cases);
	}
	
}

File: Ruby/bus.rb
=begin
 *
 *	Problem:		Bus
 *	Problem ID:	bus
 *	url:				https://open.kattis.com/problems/bus
 *	Solution:		Ruby
 *
=end

class Solver

	def solve
		number_cases = gets.chomp.to_i
		for case_number in 1..number_cases
			number_stops = gets.chomp.to_i
			result = 0
			for stop in 1..number_stops
				result = (result * 2) + 1
			end
			puts result
		end
	end
	
end

solver = Solver.new
solver.solve
